workspaces/rbac/.changeset/kind-buckets-punch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/plugin-rbac-backend': minor
```

Comment:
Don't forget to include the rbac-common plugin to the changeset

Reply:
Done. It is separated file.